diagnostic.md

```diff
@@ -15,14 +15,16 @@ Graces-MacBook-Pro:try grace$
 ```
 
 What is the full path to the current directory?
+Graces-MacBook-Pro:wdi grace$ pwd
+/Users/grace/wdi
```

Comment:
Almost.  It's /Users/grace/wdi/try since we created "try" with "mkdir try" and then entered it with "cd try".